docs/howtos/eviction-placement-disruption-budget.md

```diff
@@ -26,10 +26,11 @@ spec:
   resourceSelectors:
     - group: ""
       kind: Namespace
-      version: v1          
+      version: v1
       name: test-ns
   policy:
-    placementType: PickAll
+    placementType: PickN
+    numberOfClusters: 1
 ```
 
 The `CRP` status after applying should look something like this:
```

Comment:
Why does the crp example on line 41 and below have GEN 2?

Reply:
Looks like the GEN is 2 "magically" for a pickN CRP. Need to investigate more on it.